server/models/device.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from 'mongoose';
+
+const DeviceSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  type: {
+    type: String,
+    required: true,
+    enum: ['email', 'phone', 'sms', 'user']
```

Comment:
Yeah there were some changes that need to be made, `user` doesn't fit with our design